### PR TITLE
5817, 5841, 5842, 5843 geolocation window bug fixes

### PR DESCRIPTION
**Pull request by kellykelly3**
This PR fixes four minor issues with the Geolocation window.

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/geolocation/GeoFilterPanel.java [hunk 1]
```diff
@@ -135,7 +135,7 @@ GeoFilter getFilterState() throws GeoLocationUIException {
     private void initCheckboxList() throws TskCoreException {
         final SleuthkitCase sleuthkitCase = Case.getCurrentCase().getSleuthkitCase();
 
-        checkboxPanel.clearList();
+//        checkboxPanel.clearList();
```
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/geolocation/GeoFilterPanel.java]: remove instead of comment out

**Comment by wschaeferB** [comment]
There is a bug introduced in this PR where you get a 'case is closed' TskCoreException if you open a case and open the geolocation tool, then open a different case (the 2nd case must have waypoints, the first it does not matter) when opening the geolocation tool in the 2nd case you get the exception, this issue does not exist in the current develop branch.

**Comment by wschaeferB** [comment]
As for the refreshing the data source list, is only updating the list when the user clicks refresh after a new waypoint is added the behavior we want? Or do we want to update the the list when a data source is added or ingest is finished, currently we have different behaviors in different locations as discussed in JIRA 5720 and I just want to make sure this behavior is the behavior we desire.

**Comment by kellykelly3** [comment]
> As for the refreshing the data source list, is only updating the list when the user clicks refresh after a new waypoint is added the behavior we want? Or do we want to update the the list when a data source is added or ingest is finished, currently we have different behaviors in different locations as discussed in JIRA 5720 and I just want to make sure this behavior is the behavior we desire.

I went for consistency with CVT window.

**Pull request merged**